app/assets/javascripts/components/features/ui/components/onboarding_modal.jsx

```diff
@@ -90,26 +93,26 @@ const PageFour = ({ domain, intl }) => (
       <div className='row'>
         <div>
           <div className='figure non-interactive'><ColumnHeader icon='home' type={intl.formatMessage(messages.home_title)} /></div>
-          <p><FormattedMessage id='onboarding.page_four.home' defaultMessage='Home timeline shows posts from people you follow'/></p>
+          <p><FormattedMessage id='onboarding.page_four.home' defaultMessage='The Home timeline shows posts from people you follow'/></p>
```

Comment:
It should be either "The Home timeline" or "The home timeline." Whichever is preferable to you. The word "The" needs to be there in English for it to sound natural. The T needs to be capitalized because it's the beginning of a new sentence. Everyone already says "the home timeline" "my home timeline" "the local timeline" etc. Sometimes they'll say "on public timeline" though? Not sure why that is. But because there are these different timelines, the word "Home" "Local" "Federated" come off as descriptors, not proper nouns.

If you _really_ want to treat these as proper nouns akin to "Mastodon" versus "The Mastodon Network" then the sentence should be restructured. Written as "Home timeline shows" it doesn't come off as a proper noun. The word "Home" just looks like the regular word "home" at the beginning of a sentence. So when you read "Home timeline shows" it comes off as ungrammatical, like you forgot a word. 

So if you're attached to it being proper nouns, then it should be something like "There are several different Timelines. 
- Home Timeline shows posts from people you follow. 
- Local Timeline shows public posts from everyone on your instance. 
- Federated Timeline shows public posts from everyone who is followed by someone on your instance."

By capitalizing "Timelines" in an earlier sentence, it frames them as proper nouns. It's also important that, if they are proper nouns, that it be "Home Timeline" and not "Home timeline." With "Home timeline" the word "Home" comes off as a proper noun, but not "timeline." 

You don't say "Apple smartphone runs iOS" you say "Apple's smartphones..." or "The Apple smartphones..." (the latter sounds like an old person, but it's grammatically correct. It doesn't throw that P20 Spike in the brain of an English L1 speaker that says "This is wrong.") but if there's a proper noun called Apple Smartphone you capitalize both words. (e.g. Apple iPhone, "Apple" is a part of the name now, not a descriptor of "smartphone".)

I still think it sounds more natural not to treat them this way. "The Home timeline" or "The home timeline" feels right to me. Making "Timelines" into a proper noun feels to me like when corporations name their product "Fresh™" or "Joy™"